selfdrive/controls/lib/lateral_mpc_lib/lat_mpc.py

```diff
@@ -84,14 +86,16 @@ def gen_lat_ocp():
 
   ocp.parameter_values = np.zeros((P_DIM, ))
 
-  ocp.cost.yref = np.zeros((3, ))
-  ocp.cost.yref_e = np.zeros((2, ))
+  ocp.cost.yref = np.zeros((COST_DIM, ))
+  ocp.cost.yref_e = np.zeros((COST_DIM - 1, ))
   # TODO hacky weights to keep behavior the same
-  ocp.model.cost_y_expr = vertcat(y_ego,
-                                  ((v_ego +5.0) * psi_ego),
-                                  ((v_ego +5.0) * 4 * curv_rate))
-  ocp.model.cost_y_expr_e = vertcat(y_ego,
-                                    ((v_ego +5.0) * psi_ego))
+  costs = [
+    y_ego,
+    ((v_ego + 5.0) * psi_ego),
```

Comment:
# TODO hacky weights to keep behavior the same
Possible opportunity to get riid of this "hacky": ```+ 5.0``` ?

```suggestion
    (v_ego * psi_ego),
```